psi4/src/export_mints.cc

```diff
@@ -1011,7 +1021,7 @@ void export_mints(py::module& m)
              "Prints the molecule in Cartesians in input units adding fragment separators")
         .def("rotational_constants", &Molecule::rotational_constants,
              "Prints the rotational constants of the molecule")
-        .def("nuclear_repulsion_energy", &Molecule::nuclear_repulsion_energy,
+        .def("nuclear_repulsion_energy", &Molecule::nuclear_repulsion_energy, py::arg("dipole_field") = std::vector<double>(3,0.0),
```

Comment:
I gather C-side Mol.nre() users are required to explicitly pass in any perturbation, while py-side callers can do so or remain oblivious? Sounds fair.

Reply:
That's the status quo, yes.  If you're fine with that, I am too.  The hidden danger is that somebody coding stuff py-side may just expect external fields to work transparently with their code and it won't.  Kinda like what happened to us C-side before now...

Reply:
I wonder, can one overload `Wavefunction.molecule().nuclear_repulsion_energy()` so that it pulled the perturbation off the wfn and passed it into the mol.nre(). Would one even want to? Clearly for an un-owned Mol, a programmer couldn't expect any perturbation considerations w/o passing them explicitly.